src/Entity/User.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity()
```

Comment:
Don't forget possible indexes in this entity.

Reply:
Jep, thanks. I don't think I need any specific indexes for now, we will add them when we start searching through specific columns.